src/Lean/Widget/UserWidget.lean

```diff
@@ -56,8 +56,8 @@ private def WidgetSourceRegistry := SimplePersistentEnvExtension
     (UInt64 × WidgetSource)
     (Std.RBMap UInt64 WidgetSource compare)
 
-instance : Inhabited (Std.RBMap UInt64 WidgetSource compare) := ⟨∅⟩
-instance : Inhabited (WidgetSourceRegistry) := inferInstanceAs (Inhabited (PersistentEnvExtension _ _ (List _ × _)))
+local instance : Inhabited (Std.RBMap UInt64 WidgetSource compare) := ⟨∅⟩
```

Comment:
I think it would be better to add a global `instance : Inhabited (RBMap α β cmp) where default := .empty`.  This is useful more generally.